waffle/models.py

```diff
@@ -229,7 +229,9 @@ def is_active_for_user(self, user):
         return None
 
     def _is_active_for_user(self, request):
-        return self.is_active_for_user(request.user)
+        if user := getattr(request, "user", None):
```

Comment:
We support Python 3.5+. You cannot use the walrus operator.

Reply:
Dropped the operator. I'll look into tests later.

Thanks for your feedback.